test/integration/containerized/test_cli_containerized.py

```diff
@@ -6,43 +6,49 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 
 
+@pytest.fixture
+def skip_if_no_podman(container_runtime_installed):
+    return
+    if container_runtime_installed != 'podman':
+        pytest.skip('podman container runtime(s) not available')
+
+
 @pytest.mark.serial
-def test_module_run(cli, container_runtime_available):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(['-m', 'ping','--hosts', 'localhost', 'run', os.path.join(HERE, 'priv_data')])
+def test_module_run(cli, skip_if_no_podman):
+    r = cli(['run', '-m', 'ping','--hosts', 'localhost', os.path.join(HERE, 'priv_data')])
+    assert '"ping": "pong"' in r.stdout
 
 
 @pytest.mark.serial
-def test_playbook_run(cli, container_runtime_available):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(['run', os.path.join(HERE,'priv_data'), '-p', 'test-container.yml'])
+def test_playbook_run(cli, skip_if_no_podman):
+    r = cli(['run', os.path.join(HERE,'priv_data'), '-p', 'test-container.yml'])
+    assert 'PLAY RECAP *******' in r.stdout
+    assert 'failed=0' in r.stdout
 
 
 @pytest.mark.serial
-def test_adhoc_localhost_setup(cli, container_runtime_available, container_runtime_installed):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(
+def test_adhoc_localhost_setup(cli, skip_if_no_podman, container_runtime_installed):
+    r = cli(
         [
             'adhoc',
             '--private-data-dir', os.path.join(HERE,'priv_data'),
             '--container-runtime', container_runtime_installed,
             'localhost', '-m', 'setup'
         ]
     )
+    # TODO: look for some fact that indicates we are in container?
+    assert '"ansible_facts": {' in r.stdout
 
 
 @pytest.mark.serial
-def test_playbook_with_private_data_dir(cli, container_runtime_available, container_runtime_installed):
-    if not container_runtime_available:
-        pytest.skip('container runtime(s) not available')
-    cli(
+def test_playbook_with_private_data_dir(cli, skip_if_no_podman, container_runtime_installed):
+    r = cli(
         [
             'playbook',
             '--private-data-dir', os.path.join(HERE,'priv_data'),
             '--container-runtime', container_runtime_installed,
-            'test-container.yml'
+            'test/integration/containerized/priv_data/project/test-container.yml'  # FIXME: this may be a bug
```

Comment:
@maxamillion related to https://github.com/ansible/ansible-runner/issues/508, it would _most logically_ follow that the playbook path here is relative to the project folder from the `--private-data-dir`, but in the absence of a doc string for the argument and no prior experience with this, I cannot completely ascertain intent.